wordpress-mobile/WordPress-iOS
# Add a feature flag to control mentions in the block editor.

Adds a feature flag to control the use of mentions in the Gutenberg/Block editor.

To test:
  - Start a new post in a WP.com site
  - Check if the mention UI shows up
  - Go to MySites-> Press on the icon on the top right -> Tap App Settings -> Debug
 - Disable the Mentions in Gutenberg option
 - Go back to the editor
 - Check that the mentions button is no longer visible in the toolbar and pressing the @ in the keyboard do not activates the mention UI.
 
PR submission checklist:
